### PR TITLE
Fix/menubar

### PR DESCRIPTION
**Pull request by rfuzzo**
# menubar fixes

Implemented:
- Menubar inside titlebar
- bugfixes

<Additional notes>

### CODE REVIEW

File: WolvenKit.App/Controllers/MockGameController.cs [hunk 1]
```diff
@@ -24,35 +21,25 @@ public MockGameController()
 
         public IObservable<IChangeSet<RedFileSystemModel>> ConnectHierarchy() => _rootCache.Connect();
 
-        public List<IArchiveManager> GetArchiveManagers(bool loadmods) => new List<IArchiveManager>();
+        public List<IArchiveManager> GetArchiveManagers(bool loadmods) => new();
```
Comment (WopsS) [inline WolvenKit.App/Controllers/MockGameController.cs:24]: I know it is not related to this class or your change, but why does it have `loadmods` parameter? I find it strange that archive manager should do that.

File: WolvenKit.App/ViewModels/Documents/ScriptDocumentViewModel.cs [hunk 2]
```diff
@@ -56,13 +56,13 @@ public override async Task<bool> OpenFileAsync(string path)
         {
             _isInitialized = false;
 
-            LoadDocument(path);
+            await Task.Run(() => LoadDocument(path));
```
Comment (WopsS) [inline WolvenKit.App/ViewModels/Documents/ScriptDocumentViewModel.cs:59]: Why this change?

File: WolvenKit.App/ViewModels/Tools/ImportExportViewModel.cs [hunk 3]
```diff
@@ -619,7 +619,7 @@ private async Task ExecuteProcessAll()
                 var toBeExported = ExportableItems.ToList();
                 foreach (var item in toBeExported)
                 {
-                    success = await ExportSingle(item);
+                    success = await Task.Run(() => ExportSingle(item));
```
Comment (WopsS) [inline WolvenKit.App/ViewModels/Tools/ImportExportViewModel.cs:622]: Why `Task.Run`?

File: WolvenKit.Common/Tools/ProcessUtil.cs [hunk 4]
```diff
@@ -13,7 +13,7 @@ public static class ProcessUtil
     /// <param name="fileName"></param>
     /// <param name="arguments"></param>
     /// <returns>true if exit code is 0</returns>
-    public static async Task<bool> RunProcessAsync(string fileName, params string[] arguments)
+    public static async Task<bool> RunProcessAsync(string fileName, string arguments, string workingDir = "")
```
Comment (WopsS) [inline WolvenKit.Common/Tools/ProcessUtil.cs:16]:
```suggestion
    public static async Task<bool> RunProcessAsync(string fileName, string arguments, string workingDir = string.Empty)
```

Comment (rfuzzo) [reply WolvenKit.Common/Tools/ProcessUtil.cs:16]: "Default parameter value needs to be compile-time constant"
